### ISSUE TITLE
Pedigree visualization not showing affected status

### ISSUE THREAD

**Issue by mvelinder** [iobio/clin.iobio#214]
Should be shading/filling the circle/square for the affected individual in the pedigree

**Comment by WilliamRichards2017** [comment]
@mvelinder This is because I gave Adit a ped file with an unknown affection status. Affection status appears to be displaying properly when given a ped file with an affection status of 2.  This can be verified on dev.clin.iobio.io using the following urls, and the attatched .ped file
https://s3.amazonaws.com/iobio/samples/vcf/platinum-exome.vcf.gz
https://s3.amazonaws.com/iobio/samples/bam/NA12878.exome.bam
[testPed.txt](https://github.com/iobio/clin.iobio/files/4720039/testPed.txt)

**Comment by mvelinder** [comment]
Tried to test this, but have been stuck on 
![image](https://user-images.githubusercontent.com/22081541/83897552-f3d08380-a712-11ea-9ebd-4ef58a45102c.png) for at least 10 minutes now.

Is dev.clin working for you @WilliamRichards2017 ?

**Comment by WilliamRichards2017** [comment]
@mvelinder Yeah it's working fine for me
<img width="1421" alt="Screen Shot 2020-06-05 at 10 09 19 AM" src="https://user-images.githubusercontent.com/15776714/83898904-cdabe300-a714-11ea-8124-d0f946f089d2.png">

**Comment by adityaekawade** [comment]
Hi @mvelinder , is it working for you now? 
Did you add data for all these inputs: VCF, samples, BAM, Gene sets, PED? 
The page might be stuck if any of the above fields are missing,

**Comment by mvelinder** [comment]
@adityaekawade I've only added the VCF and then clicked Next. If we are going to require the BAM too I'd suggest keeping Next grayed out until the BAM has been loaded. 

Or we could allow them to load it without the BAM, but that sort of diminishes the "comprehensive" nature of the app. Any thoughts @AlistairNWard ?

**Comment by adityaekawade** [comment]
I have grayed out the "Next" button until both VCF and BAM are added.

**Comment by mvelinder** [comment]
That's a good enough solution for now I think @adityaekawade

**Comment by AlistairNWard** [comment]
I agree, we are trying to be comprehensive, so require all files, so this works for me. If you just want to jump in with a vcf, you can use gene.
